content/realtime/metachannels.textile

```diff
@@ -123,6 +125,60 @@ While the channel lifecycle metachannel can provide occupancy metrics, there may
 * "Inband channel occupancy events":/realtime/inband-occupancy - for use cases where the entity that needs to determine occupancy is attached to the channel. 
 * Occupancy rules - for use cases where the entity needing to determine channel occupancy is not connected to the channel. A channel occupancy reactor rule allows you to obtain occupancy updates from a set of channels in a centralized way, that doesn't involve attaching to all of them. Please "get in touch":mailto:enterprise-sales@ably.io for further information.
 
+h2(#app-stats). App statistics 
+
+The Ably system collects usage statistics on a per-app basis and exposes them via the REST API as described in the "statistics documentation":https://ably.com/documentation/general/statistics.
+
+These app statistics are also published every minute to a special metachannel named @[meta]stats:minute@, with each message being formatted according to the "app-stats JSON schema":https://schemas.ably.com/json/app-stats-0.0.1.json.
+
+You can subscribe to the stats metachannel with the following example code:
+
+```[javascript]
+  const channel = ably.channels.get("[meta]stats:minute");
+
+  channel.subscribe("update", message => {
+    data = JSON.stringify(message, undefined, 2);
+    console.log(data);
+  });
+```
+
+An example of the format of the data received is shown here:
+
+```[json]
+{
+  "name": "update",
+  "id": "trVvT-KeEw:0:0",
+  "encoding": null,
+  "data": {
+    "intervalId": "2021-10-07:16:23",
+    "unit": "minute",
+    "schema": "https://schemas.ably.com/json/app-stats-0.0.1.json",
+    "entries": {
```

Comment:
I think this should be annotated to say:

> `entries` is a flattened representation of the categories for which there are non-zero entries for this stats record.

I think somewhere - perhaps here or in the schema - there could be a more detailed definition of what these mean.

Reply:
Added `entries` to parameter table. The schema currently contains some descriptions, but not all.